docs/application/native/guides/multimedia/image-classification.md

```diff
@@ -107,7 +107,14 @@ To classify an image:
         dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
     ```
 
-4. Configure `g_engine_config` with classification model data to classify image. In the following example, TensorFlow Lite model is used and `data.tflite` and `label.txt` are in `<OwnDataPath>`. Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models):
+4. Configure `g_engine_config` with classification model data to classify image.
+   The default engine is configured by the system. You can see the supported engines in Media Vision Inference API reference (for
+		   [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html#ga0ffb25d88f8ef1f76702d9189aa6a68f)
+		   and
+		   [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html#ga0ffb25d88f8ef1f76702d9189aa6a68f).
+
+    In the following example, TensorFlow Lite model is used and `data.tflite` and `label.txt` are in `<OwnDataPath>`.
```

Comment:
1) are in the `<OwnDataPath>` folder.
2) Two continuous and, recommend to add a comma after used.

In the following example, TensorFlow Lite model is used, and `data.tflite` and `label.txt` are in the `<OwnDataPath>` folder.

or 

In the following example, TensorFlow Lite model is used, and the files `data.tflite` and `label.txt` are in the `<OwnDataPath>` folder.


@mijin85cho should the colon be added at the end of this line or is it okay to be at the end of 117. If we change here, then will have to update point no. 2 as well.

Reply:
@annie-samsung 

> @mijin85cho should the colon be added at the end of this line or is it okay to be at the end of 117.

It is ok to be at the end of line 117.

About  `<OwnDataPath>` : 
I can see you're trying to use same tone with no.2. 
However to me, `<OwnDataPath>` sound like a "path" where the required file is placed, rather than a folder.
So "`<OwnDataPath>` folder" is not wrong indeed... but not familiar to me. 
I'd like to suggest not to add folder.

@daeinki Would you let us know your opinion?

Reply:
@mijin85cho Sure, I too had the same doubt. Line 57 had the same usage, hence for uniformity sake I suggested the same.